### DISCUSSION TITLE
export to pdf

### DISCUSSION THREAD

**Discussion by dbarrosop** [maaslalani/slides#117] [Q&A] [answered]
Hello,
is it possible to export to pdf or any other format somehow? Mostly considering how to share the slides if needed.

Thanks for the awesome work!
David

**Comment by maaslalani** [comment, marked as answer]
Hey, this would be interesting although I don't know if `slides` is the best place for it. IMO sharing a markdown file is easier than sharing PDFs if both people have `slides` installed. I'm guessing your use case is sharing presentations with non-technical / non-command-line users. It may be worth looking at https://sli.dev in that case which is also markdown based.

A kind of hacky solution is that you can take a picture of each slide and then combine them into a GIF or PDF.  This is actually how I made the demo GIF in the README of this repo.

**Comment by kellpossible** [comment]
I created a script to do it using tmux, ansifilter and xelatex/beamer:

https://gist.github.com/kellpossible/a6081e945d815560be630de322fb0a1f

**Reply by patrickelectric** [reply]
@kellpossible you are a hero!
